### ISSUE TITLE
[Feature Request] Add an option to determine whether to close an issue transferred from another repo

### ISSUE THREAD

**Issue by lolimay** [offu/close-issue-app#388]
**Is your feature request related to a problem? Please describe.**  

An issue transferred from another repo may not meet the requirement in this repo.
We want the bot to ignore this issue and not close it automatically.

**Describe the solution you'd like**  

Add an option to let us control this behavior

**Describe alternatives you've considered**  

Make `exception list` more configurable, eg.

```
...
exception:
  username1:
    allow-open: true
    allow-reopen: true
  username2:
    allow-open: false
    allow-reopen: true
```

**Additional context**  
n/a

**Comment by whtsky** [comment]
Based on my rough understanding of GitHub API ( https://developer.github.com/webhooks/event-payloads/#issues ) we can't tell if the issue is transferred from another repo or not.

And I'm a bit confused about your example. How is open / reopen issue related to issue transfer?

**Comment by lolimay** [comment]
Basically, it's due to that a transferred issue doesn't meet the requirement of the target repo. So it would be closed by the bot. But what we expect is that when we reopened the issue manually, it shouldn't be closed by the bot automatically again - It's an alternative option to handle issues transferred from another repo.

**Comment by whtsky** [comment]
Exception list is implemented in #288.

Are you trying to give some maintainers reopen invalid issue permission without ”open invalid issue”  permission? That's a bit useless IMO since they can reopen the issue closed by close-issue-app.

**Comment by lolimay** [comment]
alright, it makes sense to me.
